app/assets/stylesheets/application.scss

```diff
@@ -28,6 +28,7 @@
 @import "local/member_sort";
 @import "local/error_pages";
 @import "local/member_list";
+@import "local/work_show";
```

Comment:
Did you forget to add this file to the repo?

Reply:
Yup. Oops!